_posts/2024-03-18-15.0.2-release.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Apache Arrow 15.0.2 Release"
+date: "2024-03-18 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 15.0.2 release.
+This is mostly a bugfix release that includes [**8 resolved issues**][1]
+from [**7 distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## C++ notes
+
+Several bug fixes, please see the full changelog for details.
+
```

Comment:
Can we add unexpected ABI breaking in 15.0.1 (not 15.0.2) https://github.com/apache/arrow/issues/40604 as a known problem?

Reply:
I suppose you meant on the 15.0.1 blogpost, like this? https://github.com/apache/arrow-site/pull/493/commits/edf508884c5ab19a590a1313c5082b99766dfa11

Reply:
Thanks! The 15.0.1 blog post may be better but could you also mention this in the 15.0.2 blog post? Because this was reported after 15.0.1 was released.

Reply:
Done on the 15.0.2 blog post too.

Reply:
Thanks!